vendor/modules.txt

```diff
@@ -261,7 +261,7 @@ github.com/opencontainers/go-digest/digestset
 github.com/opencontainers/image-spec/identity
 github.com/opencontainers/image-spec/specs-go
 github.com/opencontainers/image-spec/specs-go/v1
-# github.com/opencontainers/runc v1.0.0-rc92
+# github.com/opencontainers/runc v1.0.0-rc92.0.20200815064747-6dfbe9b80707
```

Comment:
Perhaps better to wait for a new release of runc? (ran into the same issue with docker, where we now temporarily disabled the new capabilities); https://github.com/moby/moby/pull/41563

Related discussion in https://github.com/opencontainers/runtime-spec/issues/1071

Reply:
@thaJeztah yes, I agree. There's a similar discussion happening here https://github.com/cri-o/cri-o/pull/4463#issuecomment-758078921